src/lib/forms/PipelineForm.svelte

```diff
@@ -19,6 +21,16 @@
     atf4_type: "",
     accuracy: "",
     is_alternative: false,
+    cost: 0,
```

Comment:
We can leave this off and not fill it out. I agree it's a bit inconsistent/confusing now, but in `types.ts` if it's optional, then we don't have to set a default. Since we collect cost with `NumberInput` and that handles undefined for us, it should work out OK